### ISSUE TITLE
Suggestion: Tweaking Banner Recipes

### ISSUE THREAD

**Issue by Guinaro** [DevOnTheRocks/RockyTweaks#6]
I relation to the other issue of anvil recipe removal, I would like to suggest the possibility to tweak banner recipes.
E.g. Currently I want to replace the iron shovel with a tconstruct:shovel to generate the shovel banner recipes.
Requested crafttweaker functionality.
Adding banner recipes, changing items with other items, removing recipes?
Possibly adding new item recipes and adding banner resources?

Guinaro

**Comment by Mohron** [comment]
I honestly have never touched the banners... Are they not crafted with the vanilla crafting table?

**Comment by Guinaro** [comment]
Yes they are, but if I do a listing of all the recipes for the crafting table with crafttweaker, all the special versions are unlisted as everything is done with nbt tags.
My crafttweakerfu isn't strong enough to tackle the mess of creating my own crafttweaker recipes for all possible variations of banner models. Let alone being able to remove them.

**Comment by Mohron** [comment]
Well I am really only focused on adding functionality that is missing from CraftTweaker, not really non-optimal situations that can be solved with some advanced CT syntax. I'm not completely ruling it out yet but I wouldn't rely on me adding this functionality.

**Comment by Guinaro** [comment]
At least thanks for taking it in consideration.

**Comment by Guinaro** [comment]
For now I have also posted a issue https://github.com/Darkhax-Minecraft/AdditionalBanners/issues/10 on the Additional Banners Mod's Github page. They are thinking about implementing a config file. So I have made a suggestion for crafttweaker support, with the option to control the banner recipes. Remove, altering, adding, ... . Maybe that mod is a better place to implement such control.

**Comment by Guinaro** [comment]
OK, this can be closed or made low priority. My crafttweakerfu has risen, and I was able to recreate the crafting recipes with regular workbench, as well as artisan worktables.

For 2 recipe script recreating all the base recipes in a workbench see this gist.
https://gist.github.com/Guinaro/b357d6e90177290967b6693741503a7d
